src/sentry/incidents/subscription_processor.py

```diff
@@ -487,11 +491,15 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
         ]
         fired_incident_triggers = []
         with transaction.atomic(router.db_for_write(AlertRule)):
+            # Triggers is the threshold - NOT an instance of a trigger
             for trigger in self.triggers:
-                if alert_operator(
+                if alert_operator(  # operator.gt/operator.lt
```

Comment:
nit: remove inline comment since it's making it a little harder to read. 
```suggestion
                if alert_operator(
```